canonical-web-and-design/ubuntu.com
# Move ESM and Livepatch to security from support

## Done

- Move `templates/esm` to `templates/security/esm`
- Move `templates/livepatch` to `templates/security/livepatch`
- Add both to `redirects.yaml`
- Add them to security secondary nav
- Remove them from support secondary nav
- Add temporary redirects for FIPS and Hardening to `/security/certifications` as requested


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check both [http://localhost:8001/security](http://localhost:8001/security) and [http://localhost:8001/support](http://localhost:8001/support) are as requested in the [brief](https://docs.google.com/document/d/1eF7P0BHUii5OQxbyr6lYz94pnQZ2pSbe_I_QuBzBkfc/edit?ts=5f57f37a#)


## Issue / Card

Fixes 
https://github.com/canonical-web-and-design/web-squad/issues/3182
https://github.com/canonical-web-and-design/web-squad/issues/3183
https://github.com/canonical-web-and-design/web-squad/issues/3184
https://github.com/canonical-web-and-design/web-squad/issues/3185
